.github/ISSUE_TEMPLATE/duplicate_bug.yml

```diff
@@ -0,0 +1,26 @@
+name: "Duplicate Bug!"
+description: "Bug is present within multiple projets"
+title: "[Duplicate Bug]: "
+labels: ["duplicate bug"]
```

Comment:
I think you can label this as "duplicate", "bug"? instead of 1?

Reply:
I would say just `duplicate` and remove the bug language altogether, just duplicate issue.

Reply:
The tags from the original issue will be copied

Reply:
okay i will do that thanks @obermillerk